packages/upload/theme/material/vaadin-upload-styles.js

```diff
@@ -96,6 +77,24 @@ registerStyles(
   { moduleId: 'material-upload' },
 );
 
+registerStyles(
+  'vaadin-upload-icon',
+  css`
+    :host {
+      display: inline-block;
```

Comment:
This rule seems to be redundant because it is already defined in `vaadin-upload-icon.js`

Reply:
Removed.